docs/self-hosted-appcircle/configure-server/_redis-domain-caution.mdx

```diff
@@ -0,0 +1,9 @@
+:::caution
+##### New Redis Subdomain Access 
+
+Upgrading from older versions to v3.18.0 or later, requires `redis` subdomain that should be registered before upgrading. For more details, you can checkout the [DNS Configuration](/docs/self-hosted-appcircle/install-server/docker.md#4-dns-settings) document section.
+
+If are not using the Appcircle server with `HTTPS`, you should also allow the Appcircle runners to connect to the Appcircle server with `TCP` `6379` port. For more details, you can checkout the [Network Access](/docs/self-hosted-appcircle/configure-server/integrations-and-access/network-access.md#appcircle-server) requirements between the Appcircle runners and the Appcircle server.
```

Comment:
Change link format:
- Remove the `docs` part
- Remove the `.md`

It should be something like that gotten from the web browser address bar. You can check other relative links in the document for the recommended style.

Reply:
Done ✅